crates/pop-common/src/lib.rs

```diff
@@ -100,6 +101,7 @@ pub fn target() -> Result<&'static str, Error> {
 ///
 /// A new Command instance configured to run the pop binary with the specified arguments
 pub fn pop(dir: &Path, args: impl IntoIterator<Item = impl AsRef<OsStr>>) -> Command {
+	#[allow(deprecated)]
```

Comment:
This seems to be way harder to get rid of, I'm afraid... the equivalent macro does not seem to work. I think we can leave it for later 😓